src/components/Navbar/Navbar.jsx

```diff
@@ -1,12 +1,14 @@
-import React from 'react';
+import { React } from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-import { HStack, Flex, Button, Image } from '@chakra-ui/react';
+import { HStack, Flex, Button, Avatar, Image, useToast } from '@chakra-ui/react';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Avatar' is defined but never used.